v2/brigadier/src/jobs.ts

```diff
@@ -20,33 +61,177 @@ export class Job {
     this.event = event
   }
 
+  /**
+   * Runs the job.
+   * 
+   * When you run the job, Brigade runs all the containers, primary and sidecar.
+   * When the primary container exits, the job ends. If sidecars are still running
+   * at this point, Brigade terminates them after a short delay.
+   * 
+   * @returns A Promise which completes when the primary container completes. If the
+   * primary container succeeded (exited with code 0), the Promise resolves; if the
+   * primary container failed (exited with nonzero code) or timed out, the Promise
+   * rejects. Sidecars do not affect success or failure of the job.
+   */
   public run(): Promise<void> {
     return Promise.resolve()
   }
 
+  /**
+   * Gets the job logs.
+   * 
+   * If the job has multiple containers, this aggregates the logs from them all.
+   */
   public logs(): Promise<string> {
     return Promise.resolve("skipped logs")
   }
+
+  /**
+   * Specifies a Runnable that consists of one or more containers. The
```

Comment:
I want to avoid 'composed' here because it sounds too much like what we're doing with groups (and possibly other composites).  Proposing "a Runnable that executes one or more containers".

Reply:
That sgtm